test-cloud/pom.xml

```diff
@@ -36,8 +36,8 @@
     <github.password/>
 
     <maven.repo.url/>
-    <maven.repo.username/>
-    <maven.repo.password/>
+    <maven.repo.username>mavenUser</maven.repo.username>
```

Comment:
Is there a reason to define those values? AFAIK they are provided as command parameters when running tests.

Reply:
No they are not neede, I'll remove them.